website/docs/r/storage_table_entity.html.markdown

```diff
@@ -0,0 +1,79 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_storage_table_entity"
+sidebar_current: "docs-azurerm-resource-storage-table-entity"
+description: |-
+  Manage an Azure Storage Table Entity.
+---
+
+# azurerm_storage_table_entity
+
+Manage an Azure Storage Table Entity.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "azuretest"
+  location = "westus"
+}
+
+resource "azurerm_storage_account" "test" {
+  name                     = "azureteststorage1"
+  resource_group_name      = "${azurerm_resource_group.test.name}"
+  location                 = "westus"
```

Comment:
since this can be interpolated; could we interpolate this value from the Resource Group?